src/main/java/com/salesforce/mirus/Mirus.java

```diff
@@ -135,10 +136,13 @@ private static void startConnect(Map<String, String> workerProps) {
     StatusBackingStore statusBackingStore =
         new KafkaStatusBackingStore(time, worker.getInternalValueConverter());
     statusBackingStore.configure(configWithClientIdSuffix(workerProps, "status"));
+    WorkerConfigTransformer configTransformer = worker.configTransformer();
```

Comment:
No need to declare `configTransformer` here, just use `worker.configTransformer()` below.